tests/delete/tests.py

```diff
@@ -263,6 +264,106 @@ def test_restrict_gfk_no_fast_delete(self):
         self.assertFalse(GenericDeleteBottom.objects.exists())
 
 
+@skipUnlessDBFeature("supports_foreign_keys")
+class OnDeleteDbTests(TestCase):
+
+    def test_db_cascade(self):
+        a = BaseDbCascade.objects.create()
+        b = RelToBaseDbCascade.objects.create(
+            name='db_cascade',
+            db_cascade=a
+        )
+        b.db_cascade.delete()
+        # BaseDbCascade is deleted
+        self.assertFalse(
+            BaseDbCascade.objects.filter(pk=a.pk).exists()
+        )
```

Comment:
Please use `self.assertIs(..., False)` instead of `self.assertFalse()` when checking for `False` rather than "a falsy value". The equivalent goes for `self.assertTrue()`.

Please update all cases of this.